addon/components/bs-modal/dialog.hbs

```diff
@@ -13,9 +13,15 @@
   {{did-insert this.getOrSetTitleId}}
   {{did-insert this.setInitialFocus}}
 >
-  <div class="modal-dialog {{this.sizeClass}} {{if @centered "modal-dialog-centered"}} {{if @scrollable "modal-dialog-scrollable"}}" role="document">
-    <div class="modal-content"
-     tabindex="-1"
+  <div class="modal-dialog {{this.sizeClass}} {{if @centered "modal-dialog-centered"}} {{if @scrollable "modal-dialog-scrollable"}} 
+  {{#if @fullscreen}}
+    {{#if (bs-eq @fullscreen true)}}
+      modal-fullscreen
+    {{else}}
+      modal-fullscreen-{{@fullscreen}}-down
+    {{/if}}
+  {{/if}}" role="document">
```

Comment:
If I recall correctly, we are currently trying to prevent linebreaks in `class` attributes even if that is causing very long lines.

```suggestion
  <div class="modal-dialog {{this.sizeClass}} {{if @centered "modal-dialog-centered"}} {{if @scrollable "modal-dialog-scrollable"}} {{#if @fullscreen}}{{#if (bs-eq @fullscreen true)}}modal-fullscreen{{else}}modal-fullscreen-{{@fullscreen}}-down{{/if}}{{/if}}" role="document">
```

Reply:
OK. Fixed

Reply:
I think you must use inline `{{if}}` in this case. https://guides.emberjs.com/release/components/conditional-content/#toc_inline-if

Reply:
Yes it was my proposal... But it seems tricky here in this case...